src/app/portlets/content-types/edit/content-types-edit.component.ts

```diff
@@ -68,7 +68,21 @@ export class ContentTypesEditComponent implements OnInit {
         ]).subscribe();
 
         this.setTemplateInfo();
-        this.dialogHeight = window.innerHeight - 50;
```

Comment:
Like I said: NEVER add a fix height to a dialog unless we know the content needs to scroll like in the iframe inside the dialog of edit page.